lib/platform.py

```diff
@@ -63,6 +69,8 @@ def have_caps(*args):
 
 def info():
   import platform
+  import distro
```

Comment:
`distro` is a non-standard python module.  Thus, please use try...except here.  And if distro is unavailable, try fallback to platform.dist or a lambda that returns "unknown"...

```python
try:
  from distro import linux_distribution as linux_dist
except:
  try:
    from platform import dist as linux_dist
  except:
    linux_dist = lambda: "unknown"
```

Reply:
did the change